### DISCUSSION TITLE
help I press the shorten url button and it stays loading

### DISCUSSION THREAD

**Discussion by johanmartinez2022** [YOURLS/YOURLS#3534] [Q&A] [answered]
I don't know why it doesn't generate short urls. I have another installation and it works perfect. I am doing this in a subdomain, the only difference is the rest of the configuration is the same

![image](https://user-images.githubusercontent.com/111463026/222931341-127ebe04-49c9-4487-a98f-2311f0b60c21.png)

**Comment by dgw** [comment, marked as answer]
https://yourls.org/docs/guide/troubleshooting/frequent-issues#endless-spinning-icon

**Reply by johanmartinez2022** [reply]
gracias
